### PR TITLE
[New module] Freesurfer registration/convert

### PR DESCRIPTION
**Pull request by anroy1**
## Describe your changes

Add the synthbet option to preproc_t1Warp conversion module - designed initially to convert freesurfer registration warp (affine and deform) to ants-itk. Then decided to generalize the module for all cases possible. Receives one affine matrix, one deform warp, source and target images for orientation (and freesurfer licence) and outputs converted affine and deform.

## List test packages used by your module

- `freeurfer.zip`
- `freesurfer_relice.zip`
- `freesurfer_transforms.zip`

## Checklist before requesting a review

- Create the tool:
  - [x] Edit `./modules/nf-scil/<category>/<tool>/main.nf`
  - [x] Edit `./modules/nf-scil/<category>/<tool>/meta.yml`
- Generate the tests:
  - [x] Edit `./tests/modules/nf-scil/<category>/<tool>/main.nf`
  - [x] Edit `./tests/modules/nf-scil/<category>/<tool>/nextflow.config`
  - [x] Add test data locally for tests with the fork repository
  - [x] Generate the test infrastructure and _md5sum_ for all outputs
- Ensure the syntax is correct :
  - [x] Check indentation abides with the rest of the library (don't hesitate to correct others !)
  - [x] Lint everything. Ensure your variables have good names.

## To Do

### CODE REVIEW

File: modules/nf-neuro/registration/convert/environment.yml
Comment (AlexVCaron) [inline modules/nf-neuro/registration/convert/environment.yml:1]: This file has changed now that we're on `nf-neuro`. You can refer to any other modules in the repo right now, I've fixed them all. Basically, this file is used for `conda` environments only, which we don't support.

We keep the file for linting, but `channels` and `dependencies` are simply empty lists, aka :

```
---
name: registration_convert
channels: []
dependencies: []
```

Replicate that for all your modules !

**Review by AlexVCaron — CHANGES_REQUESTED** [review]
Excellent work, all functionalities are there ! There is a little work to be done on syntax. To be consistent, you should stick with a naming convention. Either use `affine` or `linear` everywhere when you are talking about the affine/linear part, and drop `init` when talking about the input or output transforms.

Once adressed, LGTM !

**Review by arnaudbore — APPROVED** [review]
GTG

**Review by AlexVCaron — APPROVED** [review]

**Comment by AlexVCaron** [comment]
Great job @anroy1 ! LGTM 🚀

**Pull request merged**